src/effects/native/balanceeffect.cpp

```diff
@@ -1,7 +1,11 @@
 #include "balanceeffect.h"
 
-namespace {
+#include "util/defs.h"
 
+namespace {
+    double kMaxCorner = 22000; // Hz
```

Comment:
Why is this in an anonymous namespace? It is only referenced once for setting the maximum value of the parameter.